integration/image_create_test.go

```diff
@@ -0,0 +1,122 @@
+package integration
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/sclevine/spec"
+	"github.com/stretchr/testify/require"
+)
+
+func testImageCreate(t *testing.T, when spec.G, it spec.S) {
+	var (
+		expect *require.Assertions
+	)
+
+	it.Before(func() {
+		expect = require.New(t)
+		httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+			auth := req.Header.Get("Authorization")
+			if auth != "Bearer some-magic-token" {
+				w.WriteHeader(http.StatusUnauthorized)
+				return
+			}
+
+			var err error
+			_, err = ioutil.ReadAll(req.Body)
+			expect.NoError(err)
+
+			w.Write([]byte(imageCreateResponse))
+			return
```

Comment:
needs a default case like here (makes the test fail if there is an unhandled path): 
https://github.com/digitalocean/doctl/blob/34749b0f09bde81a41097fedd5e724a36dc7983c/integration/droplet_get_test.go#L35-L42

server should handle the path (this currently returns the same response on every call):
https://github.com/digitalocean/doctl/blob/34749b0f09bde81a41097fedd5e724a36dc7983c/integration/droplet_get_test.go#L26